spec/features/availability_spec.rb

```diff
@@ -151,8 +151,9 @@
 
   describe 'User visits catalog record page', js: true do
     it 'that has holdings to display' do
-      visit '/catalog/1839879'
-      expect(page).to have_css 'div[class="availability"][data-keys="1839879"]'
+      visit '/catalog/370199'
```

Comment:
I'm curious why the record we are testing here needed to change?

Reply:
Technically, it probably didn't. I just felt better using a record with ILLEND as the current location at least for initial testing to be sure that is displaying now, but it's subject to change once that item is returned so I guess it doesn't really matter.

Reply:
Ah okay 👍